bin/cli.js

```diff
@@ -26,6 +26,14 @@ if (args.version) {
   process.exit(0)
 }
 
+if (args.webrtc) {
+  try {
+    args.webrtc = require('electron-webrtc')({headless: true})
+  } catch (e) {
+    onerror('npm install electron-webrtc for webrtc support (use -g option if that is how you installed dat)')
```

Comment:
I'd say just recommend `npm install -g electron-webrtc`. This tailors the help message for the average user which will be using npm to install dat. Those who choose to use the development version (from git) should also know to remove the `-g` option. Also I think it'll still work if you always use `-g`